README.markdown

```diff
@@ -115,6 +131,30 @@ One can apply a "sort" constraint, which analyzes documents in the specified ord
 
     $ mongo test --eval "var collection = 'users', sort = { updated_at : -1 }" variety.js
 
+### Output formats ###
+
+Variety supports two different output formats:
+
+- ASCII: nice formatted tables (as in this readme)
+- JSON: valid JSON results for subsequent processing in other tools (see also [quiet option](https://github.com/variety/variety#quiet-option))
```

Comment:
Tomáš, I see you are linking here to https://github.com/variety/variety#quiet-option. This seems more brittle than simply linking to #quiet-option (i.e. not specifying any part of the URL other than this anchor).

But what do you think?